Formula/busco.rb

```diff
@@ -14,13 +14,17 @@ class Busco < Formula
     sha256 "d5fc62e2ab2d5cf248d6cac19de3be8c69b6ec4ebbcd8d132974dbf0add3d6ca" => :x86_64_linux
   end
 
-  depends_on "python3"
+  depends_on "python"
   depends_on "blast" => :recommended
   # Also depends on augustus and hmmer
 
   def install
+    xy = Language::Python.major_minor_version "python3"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python${xy}/site-packages"
+    system python, "import pipebricks"
+
     inreplace Dir["scripts/*.py"], "#!/usr/bin/env python", "#!#{HOMEBREW_PREFIX}/bin/python3"
-    system "python3", "setup.py", "install", "--prefix=#{prefix}"
+    system "python3", "setup.py", "install", "--user", "--prefix="
```

Comment:
I believe `--user` will install somewhere in the user's home directory. We want to use `--prefix=#{prefix}` to install in a directory that we specify.